src/index.html

```diff
@@ -1,13 +1,22 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="main">
+
+      <span class="box box--1">1</span>
+      <span class="box box--2">2</span>
+      <span class="box box--3">3</span>
+      <span class="box box--4">4</span>
+      <span class="box box--5">5</span>
+      <span class="box box--6">6</span>
+
```

Comment:
```suggestion
      <span class="box box--1">1</span>
      <span class="box box--2">2</span>
      <span class="box box--3">3</span>
      <span class="box box--4">4</span>
      <span class="box box--5">5</span>
      <span class="box box--6">6</span>
```

Ці відступи краще прибрати